websites/W/Watch Cartoon Online/presence.ts

```diff
@@ -0,0 +1,87 @@
+const presence = new Presence({
+		clientId: "936985014560755753"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let video = {
+	timeLeft: "",
```

Comment:
Shouldn't this be a number conventionally?

Reply:
It's using the format `MM:SS` and couldn't use as type `number`
~~I've tried several times with this and the presence I made before this, it just doesn't work somehow~~

Reply:
![image](https://user-images.githubusercontent.com/77577746/151735906-e9eac490-d507-4c42-a05f-1180891448e8.png)
Screenshot showing `endTimestamp` won't work if used as type `number`